pkg/descheduler/descheduler.go

```diff
@@ -107,6 +109,15 @@ func Run(ctx context.Context, rs *options.DeschedulerServer) error {
 		return nil
 	}
 
+	// Add k8s compatibility warnings to logs
+	serverVersion, serverErr := rs.Client.Discovery().ServerVersion()
+	if serverErr != nil {
+		klog.V(1).InfoS("Warning: Get Kubernetes server version fail")
+		return serverErr
+	}
+	if descheduler_minor_version != serverVersion.Minor {
```

Comment:
Shouldn't we look at the version skew? i.e. `For example descheduler v0.18 should work with k8s v1.18, v1.17, and v1.16.`

Reply:
but i check the document, it said as below, and at least what I add this logic is after descheduler 0.18 version. so for the judgement, it only check the minor version is it the same
```
“Starting with descheduler release v0.18 the minor version of descheduler matches the minor version of the k8s client packages that it is compiled with.”
```

Reply:
That's right, but we want to show the compatibility/support level which is `At this time descheduler does not have a hard dependency to a specific k8s release. However a particular descheduler release is only tested against the three latest k8s minor versions`

Reply:
thanks for notice, i already update the logic. please check :)

Reply:
if no comment i will click "resolve conversation" for this part.